src/test/java/embl/ebi/variation/commons/models/metadata/database/StudyDatabaseTest.java

```diff
@@ -40,6 +42,9 @@
     StudyRepository repository;
 
     @Autowired
+    AnalysisRepository analysisRepository;
```

Comment:
Maybe ```FileGeneratorRepository``` could be used instead of ```AnalysisRepository```? Or maybe even both, if there is any interest on a specific case.